sdv/data_processing/numerical_formatter.py

```diff
@@ -0,0 +1,101 @@
+"""Formatter for numerical data."""
```

Comment:
I wrote this in it's own file, but we could move it together with the DataProcessor, since it's the only place it's used. I don't have strong opinions either way.

Reply:
I think it's fine to leave in its own file